### PR TITLE
Change "dataxu" to "Roku"

### PR DESCRIPTION
**Pull request by GonzoJurezz**
Issue #1511

### CODE REVIEW

File: companies/roku.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    "categories": [
+        "ads"
+    ],
+    "name": "Roku International BV",
+    "runs": [
+        "dataxu"
+    ],
+    "address": "Roku International B.V.\nHerikerbergweg 238\n1101 CM Amsterdam\nThe Netherlands",
+    "email": "privacy@roku.com",
+    "web": "https://www.roku.com/de-de/",
```
Comment (baltpeter) [inline companies/roku.json:15]:
```suggestion
    "web": "https://www.roku.com/",
```

**Review by baltpeter — CHANGES_REQUESTED** [review]
Thanks for the record! I have requested some changes below.

File: companies/roku.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            "desc": "Name",
+            "type": "name",
+            "optional": false
+        },
+        {
+            "desc": "Email",
+            "type": "email",
+            "optional": false
+        }
+    ],
```
[the remark this answers is not in the document]
Comment (GonzoJurezz) [reply companies/roku.json:32]: I used their contact form as a source for the required elements. They have mentioned them in their contact form, so I thought, they need the same required elements with email.

File: companies/roku.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    "runs": [
+        "dataxu"
+    ],
+    "address": "Roku International B.V.\nHerikerbergweg 238\n1101 CM Amsterdam\nThe Netherlands",
+    "email": "privacy@roku.com",
+    "web": "https://www.roku.com/de-de/",
+    "sources": [
+        "https://docs.roku.com/published/userprivacypolicy/de/de",
+        "https://privacy.roku.com/contact",
+        "https://privacy.roku.com/contact/anonymous"
```
[the remark this answers is not in the document]
Comment (GonzoJurezz) [reply companies/roku.json:19]: https://github.com/datenanfragen/data/pull/1514#discussion_r739862632

[the remark this answers is not in the document]
Comment (GonzoJurezz) [reply companies/roku.json:19]: I wanted to hightligt, that there is another option for users with a roku account. Or should I mention it in the comments?

**Review by baltpeter — APPROVED** [review]
Thank you!

**Pull request merged**